resources/teamcity/android/keyman-android-release.sh

```diff
@@ -101,18 +85,27 @@ function do_publish() {
   export RSYNC_HOST
   export RSYNC_ROOT
 
+  if builder_has_option --fv; then
+    PUBTARGETS="app,fv"
+    export RELEASE_OEM=true
+    export RELEASE_OEM_FIRSTVOICES=true
```

Comment:
I think @mcdurdin intended the presence of these vars to determine if the child action fv is allowed.

a little info on #13725

I think mainly a precaution to prevent us accidentally publishing a local build

Reply:
I understand that, but I don't see the difference to passing `--fv`. This can be just as easily removed on TC to disable FV builds, and is less hidden than an environment variable. 

OK, I see - the env variable can be set at a higher level in TC so that it applies for multiple build configs whereas the `--fv` flag has to be changed in each build config.

Currently the `--fv` flag gets passed based on these two env variables. We could still do something similar to be able to disable FV builds in one place. But removing the hidden dependency in the TC build scripts by setting the required env variables based on a parameter has advantages IMO.